client/src/partials/price_list/pricelist.html

```diff
@@ -1,27 +1,28 @@
 <div class="flex-header">
   <div class="bhima-title">
     <ol class="headercrumb">
-      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
-      <li class="title">{{ "PRICE_LIST.TITLE" | translate }}</li>
+      <li class="static" translate> TREE.ADMIN </li>
+      <li class="active" translate> PRICE_LIST.TITLE </li>
     </ol>
-  </div>
-</div>
 
-<div class="flex-util">
-  <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
-    <span class="fa fa-plus"></span> {{ "PRICE_LIST.NEW_PRICE_LIST" | translate }}
-  </button>
+   <div class="toolbar">
+     <div class="toolbar-item">
+      <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
```

Comment:
Most of our buttons have been normal buttons.  Could you remove the `btn-sm` class to make it look the same as other modules?